.github/actions/setup_bazel_nix/action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
   rbePlatform:
     description: "RBE platform to use. If empty, RBE will not be used."
     required: false
+  nixTools:
+    description: "Nix tools to install as list of strings (e.g `['cowsay' 'ponysay']`). If empty, no tools will be installed."
```

Comment:
We should simplify the user interface for this:

```yaml
nixTools: |
    cowsay
    ponysay
```

Will give us better diffs and easier use, we can assemble the list in the action using bash.

Reply:
https://github.com/edgelesssys/constellation/actions/runs/6891198755/job/18745750123